overlord/configstate/configcore/prompting.go

```diff
@@ -36,6 +39,51 @@ import (
 
 var restartRequest = restart.Request
 
+var servicestateControl = servicestate.Control
+var serviceStartChangeTimeout = time.Minute
+
+func startHandlers(st *state.State, handlers []*snap.AppInfo) error {
+	var affectedSnaps []string
+	for _, h := range handlers {
+		affectedSnaps = append(affectedSnaps, h.Snap.InstanceName())
+	}
+
+	// start and enable prompt handlers
+	inst := &servicestate.Instruction{
+		Action: "start",
+		StartOptions: client.StartOptions{
+			Enable: true,
+		},
+
+		Scope: []string{"user"},
+	}
+
+	st.Lock()
+	// TODO hookstate? hwo to resolve conflicts
+	tts, err := servicestateControl(st, handlers, inst, nil, &servicestate.Flags{}, nil)
```

Comment:
Not sure what `tts` stands for here

Reply:
task sets :) idk why but it's fairly commonly used in the code this way

Reply:
I thought it might be task sets, but thought that would be `tss` :shrug: this is fine